### PR TITLE
 KS Vertical Divider 5.4

### PR DESCRIPTION
**Pull request by alex40724**
Feature Request:
https://www.ilias.de/docu/goto_docu_wiki_wpage_4698_1357.html

Rules:
https://github.com/ILIAS-eLearning/ILIAS/pull/909/files

### CODE REVIEW

File: src/UI/Component/Divider/Factory.php [hunk 1]
```diff
@@ -35,5 +35,28 @@
...
 	public function horizontal();
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       A Vertical Divider is used to mark a thematic change in a sequence of
+	 *       elements that are stacked from left to right.
+	 *   composition: >
+	 *     Vertical Dividers consists of a glyph-like character.
```
Comment (Amstutz) [inline src/UI/Component/Divider/Factory.php]: This is fine for me for the moment, however, I am not sure if we need a more flexible solution in future.

File: src/UI/templates/default/Divider/divider.less [hunk 2]
```diff
@@ -9,4 +9,10 @@ h6.il-divider {
 
 hr.il-divider-with-label {
 	display: none;
+}
+
+.il-divider-vertical:before {
+	content: "\2022";
+	padding-left: 5px;
```
Comment (Amstutz) [inline src/UI/templates/default/Divider/divider.less]: Use less variables here. We have something like @padding-horizontal-default.

**Review by Amstutz — APPROVED** [review]
This seems JF ready to me. If you feel ready to present, add the JF label. 

Add unit tests and improve less slightly (see comment) before merging.

**Comment by matthiaskunkel** [comment]
Jour Fixe, 12 MAR 2018: We accept this PR with the following change in the rule "purpose:  A Vertical Divider is used to mark a thematic or functional change in a sequence of elements that are stacked from left to right."

**Pull request merged**